### PR TITLE
[stable/atlantis] Fix statefulset.yaml to allow fork PRs

### PR DESCRIPTION
**Pull request by andyyaldoo**
Fix a typo for ATLANTIS_ALLOW_FORK_PRS value

#### What this PR does / why we need it:

Fix an issue where ATLANTIS_ALLOW_FORK_PRS is always empty even when `.Values.AllowForkPRs` is set to `true`

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/atlantis/templates/statefulset.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ spec:
           {{- end }}
           {{- if .Values.allowForkPRs }}
           - name: ATLANTIS_ALLOW_FORK_PRS
-            value: {{ .Values.allowForkPRS }}
+            value: "{{ .Values.allowForkPRs }}"
```
Comment (jeff-knurek) [inline stable/atlantis/templates/statefulset.yaml]: I think `{{ | quote }}` is better practice

Comment (andyyaldoo) [reply stable/atlantis/templates/statefulset.yaml]: Thanks @jeff-knurek. Updated

**Comment by adharmawan** [comment]
Hey @jeff-knurek, I've addressed your comments. Any more concerns?

**Pull request merged**